lib/manager.js

```diff
@@ -2,54 +2,101 @@
 const singleton = Symbol();
 const singletonEnforcer = Symbol();
 const Canvas = require('../app/Canvas.js');
+const { Menu, MenuItem } = require('electron').remote;
 
-let canvasStack;
+let applicationMenu;
+let canvasSelectionSubmenu;
 
-class CanvasManager {
+let canvasList;
+let currentCanvas;
+
+class AppManager {
   constructor(enforcer) {
     if (enforcer !== singletonEnforcer) {
       throw new Error('Cannot construct singleton');
     }
+    this._type = 'AppManager';
+    canvasList = new Array();
```

Comment:
I'm assuming you're reading something like this StackOverflow article: [Why is arr = [] faster than arr = new Array?](https://stackoverflow.com/questions/7375120/why-is-arr-faster-than-arr-new-array). So yes, according to that article the potential execution time savings is 20% (363ms versus 299ms). I will update the code accordingly.